app/src/main/java/org/oppia/android/app/player/exploration/BottomSheetOptionsMenuItemClickListener.kt

```diff
@@ -0,0 +1,14 @@
+package org.oppia.android.app.player.exploration
+
+/**
+ * Listener for when the user interacts with the play session options bottom sheet.
+ * [BottomSheetOptionsMenu].
```

Comment:
This looks odd. After `.` in this previous line why is `[BottomSheetOptionsMenu]` mentioned. It should be one meaningful sentence.

Reply:
Updated